__tests__/app.test.js

```diff
@@ -310,4 +336,23 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
-//here so that i can push off to 10-origin
+
+
+describe("GET /api/users", () => {
+  test("200: responds with all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((user) => {
```

Comment:
You've not asserted that this array has any contents before the forEach. If the array were empty this would pass as the expect statements will be skipped.